ReactNativeClient/lib/models/Note.js

```diff
@@ -85,13 +85,18 @@ class Note extends BaseItem {
 		if (body && body.length) {
 			const lines = body.trim().split('\n');
 			let output = lines[0].trim();
-			// Remove the first #, *, etc.
-			while (output.length) {
-				const c = output[0];
-				if (['#', ' ', '\n', '\t', '*', '`', '-'].indexOf(c) >= 0) {
-					output = output.substr(1);
-				} else {
-					break;
+			const mdLinkRegex = /!?\[(.+)\]\(.+\)/;
```

Comment:
What if the image markup is inside the body of the need note?

Reply:
So I would have to check for the image markup after removing the leading characters right?